OpenOversight/tests/test_database_cache.py

```diff
@@ -203,3 +213,51 @@ def test_documented_officers(mockdata, client, faker):
         assert has_database_cache_entry(department, KEY_DEPT_TOTAL_ASSIGNMENTS) is True
         assert has_database_cache_entry(department, KEY_DEPT_TOTAL_INCIDENTS) is True
         assert has_database_cache_entry(department, KEY_DEPT_TOTAL_OFFICERS) is False
+
+
+def test_department_counts(mockdata, client, faker):
```

Comment:
Optionally, this could probably be split into separate unit tests

Reply:
I think the bulk of the tests in that file could be, but it's at least consistent within the file. I don't have any qualms with breaking it up at a later point though.